src/__tests__/Project.test.js

```diff
@@ -52,38 +62,99 @@ describe('Project', () => {
 
     it('should be able to getDependencyGraph', async () => {
       const workspaces = await project.getWorkspaces();
-      const deps = await project.getDependencyGraph(workspaces);
-      const { valid, graph } = deps;
+      const { valid, graph } = await project.getDependencyGraph(workspaces);
+      const expectedDependencies = {
```

Comment:
Refactored all these tests to be more explicit as I managed to change an implementation at one point, without breaking a test.